pom.xml

```diff
@@ -94,7 +94,7 @@
         <rocketmq.version>4.9.3</rocketmq.version>
```

Comment:
We should upgrade to version 5.1.0 to support copying messageType from TopicConfig to TopicConfigInfo.
By the way, messageType should come from the attributes

Reply:
@Oliverwqcwrw  we donot need this **private String messageType** we need member variable of type Map right ?

Reply:
we need messageType in the TopicConfigInfo, you can get the messageType form the map of the TopicConfig directly

Reply:
> we need messageType in the TopicConfigInfo, you can get the messageType form the map of the TopicConfig directly

got your point

Reply:
> We should upgrade to version 5.1.0 to support copying messageType from TopicConfig to TopicConfigInfo. By the way, messageType should come from the attributes

updating version to 5.1.0 in pom.xml is throwing lot of error 
![image](https://user-images.githubusercontent.com/43075376/225405938-610adc90-a4a5-40f5-8e29-1f515537202f.png)

Reply:
You can try to fix it,
it seems the class location has changed

Reply:
@Oliverwqcwrw I have fixed most of the issues but there is some new methods added in MQAdminExtImpl.java that is unimplmented in rocketmq-dashboard what to do with this? Please check below image for ref
![image](https://user-images.githubusercontent.com/43075376/225528060-bc77aa52-56d1-43ca-b548-e64f93793cc1.png)

Reply:
Because there are many feature after 5.0, so you can see many unimplement function,

IMO, you can give an empty implementation or implement it if you have enough time.

BTW, it is better to split this pr because it has many changes.

Reply:
for which of the changes/commit I have to create diff PR ?

Reply:
How about upgrading the version to 5.1.0 and adding a MessageType attribute?